app/renderer/lobby/reducers/session.ts

```diff
@@ -64,7 +65,8 @@ const initialState: ISessionState = {
   playback: PlaybackState.Idle,
   startTime: new Date().getTime(),
   secret: '',
-  serverClockSkew: 0
+  serverClockSkew: 0,
+  maxUsers: DEFAULT_USERS_MAX
```

Comment:
Update the value of `DEFAULT_MAX_USERS` to the value decided for no limit (`MAX_USERS_INFINITE`).